lib/services.identity.js

```diff
@@ -516,11 +516,6 @@ api._createIdentity = function(options, req, callback) {
     }
     // result details
     var details = {identity: results.createIdentity};
-    // schedule identity created event
-    bedrock.events.emitLater({
-      type: 'bedrock.Identity.created',
```

Comment:
It does get used -- it's in the config for sending an email when a user joins:

https://github.com/digitalbazaar/bedrock-idp/blob/master/lib/config.js#L33-L52

We should probably keep it for now, but add a listener to the `postInsert` event and emit it there. We can reevaluate removing it later.